docs/src/cluster/turbine-block-propagation.md

```diff
@@ -32,12 +32,22 @@ Finally, the following diagram shows a two layer cluster with a fanout of 2.
 
 ![Two layer cluster with a Fanout of 2](/img/data-plane.svg)
 
+### Neighborhoods
+
+The following diagram shows how two neighborhoods in different layers interact. To cripple a neighborhood, enough nodes \(erasure codes +1\) from the neighborhood above need to fail. Since each neighborhood receives shreds from multiple nodes in a neighborhood in the upper layer, we'd need a big network failure in the upper layers to end up with incomplete data.
+
+![Inner workings of a neighborhood](/img/data-plane-neighborhood.svg)
+
 ### Configuration Values
 
 `DATA_PLANE_FANOUT` - Determines the size of layer 0. Subsequent layers grow by a factor of `DATA_PLANE_FANOUT`. The number of nodes in a neighborhood is equal to the fanout value. Neighborhoods will fill to capacity before new ones are added, i.e if a neighborhood isn't full, it _must_ be the last one.
 
 Currently, configuration is set when the cluster is launched. In the future, these parameters may be hosted on-chain, allowing modification on the fly as the cluster sizes change.
 
+The Redundant Turbine Path feature controls whether or not shreds are sent between nodes in the same neighborhood. When enabled, there is no propagation of shreds between nodes in the same neighborhood. When disabled, a single anchor node (rotated for each shred) is responsible for forwarding shreds to every node in its neighborhood.
```

Comment:
Should this reference the feature explicitly?
`4Di3y24QFLt5QEUPZtbnjyfQKfm6ZMTfa6Dw1psfoMKU | inactive                | NA              | drop redundant turbine path`

Reply:
Good question.. I've been mulling over what the right thing is to do here.

The docs should be high level, so getting into the weeds on feature IDs feels wrong. However, the high level architecture is significantly different depending on this feature, so not mentioning it at all seems wrong as well.

I'm open to whatever y'all think is best.

Reply:
Maybe add a notes section at the end of the doc with the feature ID and name and an anchor link to the ID from this section? I think it's worth calling out given the behavior change and the general level of detail in the doc.

Reply:
Alternatively, it may be better to just describe the existing behavior. You could add a comment to the feature (#24352) to update the docs with the new text once enabled.

Reply:
The drop-redundant-path feature is right now [head of the feature activation queue](https://github.com/solana-labs/solana/wiki/Feature-Gate-Activation-Schedule#current-schedule) and will be activated as soon as mainnet moves to v1.14. So maybe ~2 weeks from now.

Given that, I am thinking maybe it is not worth documenting the old behavior at all.
We can either add a note this is the current testnet design which will be rolled out on mainnet soon, or hold on merging this until the feature is activated.

Reply:
I think you're right. Best to only describe how MNB actually behaves and update the docs when the feature gets enabled.